app/helpers/date_filter_helper.rb

```diff
@@ -27,4 +30,25 @@ def total_responses_header(total_count, from, to)
       response_total
     end
   end
+
+  def date_range_track_action(from_date=nil, to_date=nil)
```

Comment:
As mentioned in person, it's probably worth extracting `date_range_track_action` and `date_range_track_label` into their own class (eg something like):

```ruby
class DateRangeTrackEvent
   def initialize(from_date, to_date)
     ...
   end

  def action
    ...
  end

  def label
    ...
  end
end
```